recipes/server_conf.rb

```diff
@@ -22,7 +22,7 @@
   owner "postgres"
   group "postgres"
   mode 0600
-  notifies change_notify, 'service[postgresql]', :delayed
+  notifies change_notify, 'service[postgresql]', :immediately
```

Comment:
Oh. I see you have very descriptive commit messages. I'll look at those more closely and they might answer my current questions.